File: JavaScript/app/models/spock.js
var Shape = require('./shape');

var Spock = Object.create(Shape);

Spock.setup = function (ShapeType) {
	this.init({
		shapeType: ShapeType.spock,
		winsTo: [ShapeType.rock, ShapeType.scissor]
	});
}

module.exports = Spock;

File: NET/Kata.RockPaperScissor/Spock.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kata.RockPaperScissor
{
    public class Spock : Shape
    {
        public Spock()
        {
            this.ShapeType = ShapeType.Spock;
            Name = "Spock";
            winsTo = new List<ShapeType>() { ShapeType.Rock, ShapeType.Scissor };
        }
    }
}
